examples/petsc/bps.c

```diff
@@ -468,6 +468,15 @@ int main(int argc, char **argv) {
       num_ceed_resources = 1;
     }
   }
+  if ((bp_choices[0] == CEED_BP5 || bp_choices[0] == CEED_BP6) && (rp->simplex)) {
+    // Set up libCEED
+    Ceed ceed;
+    CeedInit(ceed_resources[0], &ceed);
+    // LCOV_EXCL_START
+    return CeedError(ceed, CEED_ERROR_UNSUPPORTED,
```

Comment:
This is in a PETSc driver, so I'd use `SETERRQ()`

Reply:
Thanks, fixed